PayButton/paybutton-server
# feat: sideshift ws endpoint

Depends on
---
- [x] https://github.com/PayButton/paybutton/pull/419

Description
---

Adds logic to the websocket server to support SideShift integration in the PayButton client.

Test plan
---

1. Create an account in http://sideshift.ai
2. Put your SideShift affiliate id in the `paybutton-config.json` file, e.g:
"sideshiftAffiliateId": "ABc123DfG"
3. Put your SideShift secret in the `.env.local` file, e.g:
    SIDESHIFT_SECRET_KEY=my123secret456account890
4. Restart the containers

5. Follow along the next steps in the client using  at https://github.com/PayButton/paybutton/pull/419

6. Make sure entries are being created at the table `SideshiftShift` everytime a client creates a shift (every time a deposit address is available)
